Add feature to test Helm charts (#80)
giantswarm/architect-orb: CHANGELOG.md

### Added

> - Introduce `helm-lint` which uses Helm chart testing to lint Helm Charts

Reviewer: I don't consider commands to be real interface of the orb. Let's mention changes to the job instead which is important from the user perspective.

```suggestion
- Introduce Helm Chart testing and linting in `push-to-app-catalog` job.
```
